types/eventsource/lib/eventsource-polyfill/index.d.ts

```diff
@@ -1,16 +1,17 @@
 declare class EventSource {
   constructor(url: string, eventSourceInitDict?: EventSource.EventSourceInitDict);
-
-  static CLOSED: EventSource.ReadyState;
```

Comment:
`CLOSED`, `CONNECTING`, and `OPEN` are available on both the constructor (`static`) and the prototype.